docker-compose.yml

```diff
@@ -39,9 +40,11 @@ services:
       context: .
     image: frocket/local-lambda:latest
     container_name: mock-lambda
+    volumes:
+      - ./data:/var/task/data:ro,cached
     environment:
       - FROCKET_REDIS_HOST=redis
-      - FROCKET_REDIS_PORT=6380
+      - FROCKET_REDIS_PORT=${FROCKET_REDIS_PORT:-6379}
       - FROCKET_S3_AWS_ENDPOINT_URL=http://mock-s3:9000
       - FROCKET_S3_AWS_ACCESS_KEY_ID=testonly
       - FROCKET_S3_AWS_SECRET_ACCESS_KEY=testonly
```

Comment:
We can move all of these to a `.env` file or use compose vars if you like since they're repeating, I can do that if you want

Reply:
Let's consider